src/main/java/com/ecwid/maleorang/method/v3_0/lists/StatsInfo.kt

```diff
@@ -59,7 +59,7 @@ class StatsInfo : MailchimpObject() {
 
     @JvmField
     @Field
-    var open_rate: Int? = null
+    var open_rate: Double? = null
```

Comment:
Well... I suppose it couldn't hurt.  I'll change those and post another pull request.